### PR TITLE
Prepare Azure Core for August Release

### PR DESCRIPTION
**Pull request by alzimmermsft**

### CODE REVIEW

File: sdk/core/azure-core-management/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-08-07)
```
Comment (samvaity) [inline sdk/core/azure-core-management/CHANGELOG.md:3]: Would this also be released as part of core release pipeline?

Comment (alzimmermsft) [reply sdk/core/azure-core-management/CHANGELOG.md:3]: Yup!

**Comment by alzimmermsft** [comment]
Overriding check-enforcer as a flaky test happened in Storage CI, will file a follow-up issue to begin tracking these.

**Comment by weshaggard** [comment]
>Overriding check-enforcer as a flaky test happened in Storage CI, will file a follow-up issue to begin tracking these.

Please make sure there is an issue tracking the flakey storage test.

**Comment by alzimmermsft** [comment]
> 
> 
> > Overriding check-enforcer as a flaky test happened in Storage CI, will file a follow-up issue to begin tracking these.
> 
> Please make sure there is an issue tracking the flakey storage test.

I should have phrased this better, Storage itself doesn't have flaky tests but libraries transitively being build in its `from source` runs do. I'll get an issue filed soon.

**Comment by alzimmermsft** [comment]
@weshaggard @JimSuplizio here is the issue giving a quick overview of what was failing, I'm going to go through previous runs to add examples.

https://github.com/Azure/azure-sdk-for-java/issues/13980

**Pull request merged**